src/Core/IToolWindowService.cs

```diff
@@ -43,5 +43,7 @@ public interface IToolWindowService
         /// </para>
         /// </remarks>
         void EnsureToolWindowExists(Guid toolWindowId);
+
```

Comment:
Should I have added a comment? (Just because the other methods in the class already have them)

Reply:
Yes - something like:

```
        /// <summary>
        /// Returns a tool window of the specified type, or null if one could not be found.
        /// </summary>
        /// <typeparam name="T">The type of tool window. Should be a subclass of <see cref="Microsoft.VisualStudio.Shell.ToolWindowPane"/></typeparam>
```

In general, all interfaces should have comments, on the interface and on the methods.

We don't generally add comments on the classes that implement the interface, unless there is something implementation-specific that we want to capture.